src/main/webapp/index.html

```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Google Dashboard</title>
-</head>
+<meta charset="UTF-8">
+<title>Google Dashboard</title>
+<link rel="stylesheet" href="styles.css">
 <body>
```

Comment:
Fun fact - body is also an optional tag. You can remove the opening and closing body tags.

Reply:
Didn't know that - will do and thanks!

Reply:
`RESOLVED`